cmd/main_postsubmit.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
```

Comment:
What you've done here looks like it will work just fine, but I think a bit more time should be put into design here for a couple reasons. First I think this will probably grow and second we have a little breathing room right now and we should start some good design habits before the rush comes.

This could benefit from some object oriented design I think so we have some separation of concerns better defined. For example the main noun we are dealing with here seems to be "Command". Something like:
```
type Command struct {
   releaseBranch string
   gitRoot string
   release string
   artifactBucket string
   args []string
}

func NewCommand() *Command {
    cmd := new(Command)
    cmd.releaseBranch = os.Args[2]
    cmd.release = os.Args[3]
    cmd.artifactBucket = os.Args[11]
    gitRootOutput, err := exec.Command("git", "rev-parse", "--show-toplevel").Output()
    if err != nil {
        log.Fatalf("There was an error running the git command: %v", err)
    }
    gitRoot := strings.Fields(string(gitRootOutput))[0]
    fmt.Println(gitRoot)
    cmd.args = []string{"-C", gitRoot + "/projects/", os.Args[1],
        "RELEASE_BRANCH=" + os.Args[2],
        "RELEASE=" + os.Args[3],
        "DEVELOPMENT=" + os.Args[4],
        "AWS_REGION=" + os.Args[5],
        "AWS_ACCOUNT_ID=" + os.Args[6],
        "BASE_IMAGE=" + os.Args[7],
        "IMAGE_REPO=" + os.Args[8],
        "GO_RUNNER_IMAGE=" + os.Args[9],
        "KUBE_PROXY_BASE_IMAGE=" + os.Args[10],
        "IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)'"}
    return cmd
}
```